### ISSUE TITLE
[a11y]: [svg, Button, TextInput ] A11y (Accessbility) violation

### ISSUE THREAD

**Issue by loftywaif002** [carbon-design-system/carbon#18854]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

v1.76.0

### React version

18.2.0

### Automated testing tool and ruleset

IBM Accessibility Equal Access Toolkit: Accessibility Checker

### Assistive technology

No

### Description

Hi Carbon team,

As part of an IBM-wide requirement, a requirement is for DataStage UI (DataStage/ds-canvas) to be accessibility compliant, via the [IBM Equal Access Accessibility Checker tool](https://chromewebstore.google.com/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp?hl=en-US).

Screenshot: 

![Image](https://github.com/user-attachments/assets/5f9c096b-3656-4399-bf9c-997b4c0c7907)

Violation Report details : 

[Accessibility_Report-cff-test-flow - IBM Cloud Pak for Data.xlsx](https://github.com/user-attachments/files/19331330/Accessibility_Report-cff-test-flow.-.IBM.Cloud.Pak.for.Data.xlsx)

### Reproduction/example

@carbon/react

### Steps to reproduce

1. Create a ETL Flow using transformer node using https://www.ibm.com/products/datastage using chrome browser
For example: RowGen node -> Transformer Node ->  Peek node
2. Add 1 column on Rowgen 
3. Double Click on transformer node
4. If new transformer toggle button is not on, turn it on as it is highlighted below and open the left side panel

![Image](https://github.com/user-attachments/assets/8b7dccfd-b34d-4601-90ea-c41d02f4fa6c)

5. Assuming Accessibility checker tool is installed Tool Link: Equal Access checker [https://github.com/IBMa/equal-access]

Run the accessibility checker against the new transformer with left side panel open. There should be 23 violations for accessibility.

### Suggested Severity

Severity 3 = User can complete task, and/or has a workaround within the user experience of a given component.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by alisonjoseph** [comment]
Hi, thanks for reporting this, do you mind recreating in a [Stackblitz](https://stackblitz.com/github/carbon-design-system/sandboxes/tree/main/react/latest/react-19-vite/?preset=node=) as we're not seeing this issue in Storybook.

**Comment by 2nikhiltom** [comment]
Hey @loftywaif002 could you recreate this issue on a [Stackblitz](https://stackblitz.com/github/carbon-design-system/sandboxes/tree/main/react/latest/react-19-vite/?preset=node=) for us to debug and fix.
